src/python/WMCore/WMBS/MySQL/Jobs/LoadForErrorHandler.py

```diff
@@ -19,25 +19,16 @@ class LoadForErrorHandler(DBFormatter):
     to determine ACDC records for skipped files in successful jobs, this mode
     is used when a file selection is provided.
     """
-    sql = """SELECT wmbs_job.id, jobgroup, wmbs_job.name AS name,
-                    wmbs_job_state.name AS state, state_time, retry_count,
-                    couch_record,  cache_dir, wmbs_location.site_name AS location,
-                    outcome AS bool_outcome, fwjr_path AS fwjr_path, ww.name as workflow,
-                    ww.task as task, ww.spec as spec, wmbs_users.owner as owner,
-                    wmbs_users.grp as grp
+    sql = """SELECT wmbs_job.id, wmbs_job.jobgroup,
+               ww.name as workflow, ww.task as task
              FROM wmbs_job
                INNER JOIN wmbs_jobgroup ON wmbs_jobgroup.id = wmbs_job.jobgroup
                INNER JOIN wmbs_subscription ON wmbs_subscription.id = wmbs_jobgroup.subscription
                INNER JOIN wmbs_workflow ww ON ww.id = wmbs_subscription.workflow
-               INNER JOIN wmbs_users ON ww.owner = wmbs_users.id
-               LEFT OUTER JOIN wmbs_location ON
-                 wmbs_job.location = wmbs_location.id
-               LEFT OUTER JOIN wmbs_job_state ON
-                 wmbs_job.state = wmbs_job_state.id
              WHERE wmbs_job.id = :jobid"""
 
-    fileSQL = """SELECT wfd.id, wfd.lfn, wfd.filesize size, wfd.events, wfd.first_event,
-                   wfd.merged, wja.job jobid, wpnn.pnn
+    fileSQL = """SELECT wfd.id, wfd.lfn, wfd.filesize AS size, wfd.events, wfd.first_event,
```

Comment:
I thought I could use the same syntax for oracle, but it actually complains about "ORA-00923: FROM keyword not found where expected", I'm rolling back that change on the oracle class.